lms/models/users.py

```diff
@@ -26,3 +31,40 @@ def auth(username: str, password: str) -> User:
     elif user.role.is_unverified:
         raise ForbiddenPermission(_('עליך לאשר את מייל האימות'), 403)
     return user
+
+
+def generate_confirmation_token(user: User) -> str:
+    return SERIALIZER.dumps(user.mail_address, salt=retrieve_salt(user))
+
+
+def generate_session_token(username: str, password: str) -> str:
+    return SERIALIZER.dumps([username, password])
+
+
+def send_confirmation_mail(user: User) -> None:
```

Comment:
I don't think we should place these under `/models`, as `models` is a bridge between the DB and the view